coderedcms/templates/wagtailcore/password_required.html

```diff
@@ -0,0 +1,20 @@
+{% extends "coderedcms/pages/web_page_notitle.html" %}
+{% load i18n bootstrap4 %}
+
+{% block content %}
+<div class="container">
+    <div class='text-center mt-5'>
+        <h3>{% trans "Password Required" %}</h3>
+        {% block password_required_message %}
+            <p>{% trans "You need a password to access this page." %}</p>
+        {% endblock %}
+        <form action="{{ action_url }}" method="POST">
+            {% csrf_token %}
+            <div class="form-group">
+                {% bootstrap_form form layout='horizontal' %}
```

Comment:
I think the `bootstrap_form` call should go outside of a `form-group` div... I believe in this case it would be generating nested `form-group` divs which is not recommended bootstrap syntax.